.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/pybind/pybind11.git
 [submodule "libCZI"]
 	path = libCZI
-	url = https://github.com/zeiss-microscopy/libCZI.git
+	url = https://github.com/ZEISS/libczi.git
```

Comment:
This obviously works. An alternative is to use CMake's fetch_content methodology to pull the resource. In this case I don't think either has strong advantages.